test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/TxnVerbs.java

```diff
@@ -330,8 +352,58 @@ public static HapiContractUpdate contractUpdate(String contract) {
 		return new HapiContractUpdate(contract);
 	}
 
+	public static HapiSpecOperation contractDeploy(final String contractName, final Object... params) {
+		return withOpContext((spec, ctxLog) -> {
+			final var path = String.format(RESOURCE_PATH + ".bin", contractName);
+			validateFileExists(path);
+			final var payer = cryptoCreate("PAYER");
```

Comment:
Do we always want to create a new hedera account for each contract deploy?
Also, what if I want to create the contract with some specific account (e.g. use Alice as sender to the contract create hedera txn)?
And what if I want to test create with different amounts of `gas`?

Reply:
Solved.